files/run_tests.sh

```diff
@@ -2,6 +2,7 @@
 
 set -x
 
+# shellcheck source=/dev/null
 source /src/files/setup_env_in_openshift.sh
```

Comment:
huh, is this telling shellcheck to not try to access that file?

Reply:
Yes, there's no such file in the repository (it is in the image/container) so shellcheck can't access it.
Should I add a comment with a link to https://www.shellcheck.net/wiki/SC1091 ?

Reply:
it would definitely be welcoming to people who see this line for the first time, thank you for the explanation